src/main/java/ua/lviv/iot/algo/part1/lab1/controller/HornetController.java

```diff
@@ -0,0 +1,47 @@
+package ua.lviv.iot.algo.part1.lab1.controller;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ua.lviv.iot.algo.part1.lab1.models.Hornet;
+import ua.lviv.iot.algo.part1.lab1.service.HornetService;
+
+@RestController
+@RequestMapping(path = "/Hornets")
+public class HornetController {
+
+
+    private final HornetService hornetService;
+
+    @Autowired
+    public HornetController(HornetService hornetService) {
+        this.hornetService = hornetService;
+    }
+
+    @GetMapping(path = "/get/{id}")
+    public Hornet getHornet(@PathVariable long id) {
+        return hornetService.findById(id);
+    }
+
+    @RequestMapping(path = "/post")
+    @PostMapping()
+    public void postHornet() {
+        var hornet = new Hornet();
+        hornetService.addHornet(hornet);
+    }
+
+    @RequestMapping(path = "/edit")
```

Comment:
please use REST approach for path creation as it is explained here: https://youtu.be/lq2NSBTxp_8?t=1676